snippets/ptp_PtpConfigMaster.yaml

```diff
@@ -0,0 +1,126 @@
+# The grandmaster profile is provided for testing only
```

Comment:
Why is this YAML needed in the docs repo if it's for testing-purposes only?

Reply:
The YAML is pulled directly from RAN ZTP source. I'll update the YAML to remove this line.